salt/modules/pkgin.py

```diff
@@ -214,7 +227,7 @@ def version(*names, **kwargs):
     return __salt__['pkg_resource.version'](*names, **kwargs)
 
 
-def refresh_db():
+def refresh_db(force=False):
     '''
     Use pkg update to get latest pkg_summary
 
```

Comment:
Can you add documentation for the new `force` option here? (Don't forget to add a `.. versionadded:: Oxygen` tag, too. :)

Reply:
Done